exec/java-exec/src/main/java/org/apache/drill/exec/rpc/user/UserSession.java

```diff
@@ -160,6 +173,14 @@ public int getQueryCount() {
     return queryCount.get();
   }
 
+  public String getSessionId() {
+    return sessionId;
+  }
+
+  public UserSessionContextInformation getSessionContextInfo() {
+    return sessionContextInfo;
+  }
```

Comment:
As above, will we return null if user didn't set session context info in builder?